src/azure-cli-core/azure/cli/core/util.py

```diff
@@ -230,11 +230,38 @@ def read_file_content(file_path, allow_binary=False):
     raise CLIError('Failed to decode file {} - unknown decoding'.format(file_path))
 
 
+def preprocess_json(original):
+    lines = original.splitlines()
+    for line_index, line in enumerate(lines):
+        if not ("#" in line or "//" in line):
```

Comment:
What about /* */ comments?  Powershell allows them, so I assume the portal is handling them.

Reply:
i could add code to process "/*" comments as well after json is concatenated.